css/main.css

```diff
@@ -1,3 +1,209 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+=======
+s
+
+/* General */
+body {
+  font-family: Quicksand Light 300;
+  font-weight: 300;
+  color: #2f2f2e;
+}
+
+.title-txt {
+  font-family: Libre Baskeville;
+  font-size: 65px;
+}
+.nor-txt {
+  font-size: 20px;
+  font-weight:300;
+}
+
+.mid-txt {
+  font-size: 30px;
+  text-align: center;
+  font-weight: 300;
+  font-style: italic;
+} 
+
+.standard-btn {
+  background-color: #cff4ec;
+  border: 0px;
+  padding-bottom: 10px;
+  padding-top: 10px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.heading {
```

Comment:
You could simply your code and remove the need to add the `heading` class to all your h1 tags. Define the styles on the h1 tag instead.

E.g.
h1 {
  font-family: Libre Baskeville;
  font-size: 55px;
}